packages/ethereum/deployments/default/xdai/HoprChannels.json

```diff
@@ -1,7 +1,5 @@
 {
-  "address": "0xA1Ee710b6C234Df1CF52Dc619Eec6f9A4685d662",
-  "transactionHash": "0x4f358125fa34c451de744b7bdac3d50280d89e5fdd2ffc2c6ed150e4499d41ea",
-  "blockNumber": 17739880,
```

Comment:
same here, we should neither drop blockNumber nor transactionHash

Reply:
It's not me, it's CI pushing stupid things. Let's deal with this later.